NEWS.md

```diff
@@ -32,6 +32,7 @@
 
 ### Features
 * Added new column complete_updated_date into INSTANCE table that will be used in mod-oai-pmh module: [MODINVSTOR-1105](https://issues.folio.org/browse/MODINVSTOR-1105)
+* Added insert script for single instance and holdings , that will be used in mod-dcb : [MODINVSTOR-1102](https://issues.folio.org/browse/MODINVSTOR-1102)
```

Comment:
This version has already released, please move this line to v27.1.0 release text block

Reply:
fixed